include/lsst/meas/extensions/shapeHSM/HsmMomentsControl.h

```diff
@@ -246,8 +257,9 @@ class HsmPsfMomentsDebiasedAlgorithm : public HsmPsfMomentsAlgorithm {
 
     typedef std::shared_ptr<const HsmPsfMomentsDebiasedControl> Control;
 
-    HsmPsfMomentsDebiasedAlgorithm(Control const & ctrl, std::string const & name, afw::table::Schema & schema) :
-        HsmPsfMomentsAlgorithm(ctrl, name, schema) {
+    HsmPsfMomentsDebiasedAlgorithm(Control const & ctrl, std::string const & name, afw::table::Schema & schema,
+                                   char const* doc = "Debiased PSF adaptive moments algorithm from HSM") :
```

Comment:
As above, "Debiased PSF adaptive moments via the HSM shape algorithm", or similar.

Reply:
and this to "Debiased adaptive moments of the PSF via the HSM shape algorithm".